### ISSUE TITLE
How the representative points can be learned?

### ISSUE THREAD

**Issue by zosel260** [zhanght021/RPT#14]
Dear Zhanght,
Thank you for sharing your excellent codes.
I read the RPT paper.  I'm curious about the how the representative points can be learned that indicates the object boundaries and semantically prominent regions.
According to my understand from the paper, the training steps for the points are like below:

1. model estimates the positions of the points. 
2. sudo box is obtained from points using min/max operations
3. overlap with ground-truth bbox is computed.
4. overlap-related loss is computed so model can learn from this loss.

Because there is only bbox overlap loss based on min/max operation, I guess the points could be  moved to the bbox boundaries rather than object boundaries or semantically prominent positions.
I don't know how the points can be  moved to the object boundaries and  semantically prominent positions without other loss.
Thanks in advance.

**Comment by zhanght021** [comment]
hello, our idea comes from reppoints(RepPoints: Point Set Representation for Object Detection)(https://arxiv.org/abs/1904.11490), probably you can refer to it.
I admit that the output points often don't actually look like they are key points. And i also draw the output of reppoints, it also cannot represent the key points.

 In order to make better use of the information of key points， we update the RPT to RPT++, but the paper is under review， we will publish the paper later.

**Comment by zosel260** [comment]
Thank you for kind replay.

**Comment by laisimiao** [comment]
@zhanght021 Is the paper on axriv?(I refer to RPT++)
thank you very much
